cfme/containers/provider/__init__.py

```diff
@@ -584,6 +584,10 @@ def is_displayed(self):
         # We use 'in' for this condition since when we use search it'll include (Names with "...")
         return self.SUMMARY_TEXT in self.summary.text
 
+    @property
+    def table(self):
+        return self.entities.elements
```

Comment:
entities is Conditional widget here and this property is dependent on a potential state of the condition. 
https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L3248

We should not define a property in this case.

In this case, the test is what needs to be fixed. 

Example for getting the column rows on the provider all view:
```
In[29]: view.entities.elements.headers
Out[29]: (None, None, u'Name', u'Hostname', u'IP Address', u'Port', u'Type')
```